### DISCUSSION TITLE
Show in the carousel that the selected collection is empty

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#19818] [Ideas]
Just for consistency, I would suggest showing a message like `This collection is empty` to better explain to the user why there are no maps in the carousel. This is what you can currently see:
![image](https://user-images.githubusercontent.com/43073074/185104271-9adb4e70-b43a-4d3e-a321-a888d5cf3adc.png)

**Comment by peppy** [comment]
A clause can be added to the placeholder that lets you deselect the current collection, similar to the other lines that are displayed there.
